src/net/rpc.rs

```diff
@@ -1609,8 +1609,19 @@ impl ConversationHttp {
         estimated_len: u64,
     ) -> Result<(), net_error> {
         let response_metadata = HttpResponseMetadata::from(req);
+        let tip = SortitionDB::get_canonical_burn_chain_tip(sortdb.conn())?;
+        let stacks_epoch = sortdb
```

Comment:
Can't this just be `SortitionDB::get_stacks_epoch(tip.block_height)`?  Why involve an indexed connection, which potentially has a MARF read?